src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -202,6 +203,7 @@ const RootStack = createCustomStackNavigator<AuthScreensParamList>();
 
 const modalScreenListeners = {
     focus: () => {
+        TooltipManager.cancelPendingAndActiveTooltips();
         Modal.setModalVisibility(true);
     },
```

Comment:
And because the RHP modal uses `setModalVisbility`, we need to manually cancel it here.